governance/governance.go

```diff
@@ -780,23 +773,8 @@ func (w *GovernanceWorker) RecordReply(proposal *citizenscientist.Proposal, prot
 				glog.Errorf("Error: %v", err)
 			}
 			envAdds[config.ENVVAR_PREFIX+"AGREEMENTID"] = proposal.AgreementId
-			envAdds[config.COMPAT_ENVVAR_PREFIX+"AGREEMENTID"] = proposal.AgreementId
-			envAdds[config.ENVVAR_PREFIX+"CONTRACT"] = w.Config.Edge.DVPrefix + proposal.AgreementId
-			envAdds[config.COMPAT_ENVVAR_PREFIX+"CONTRACT"] = w.Config.Edge.DVPrefix + proposal.AgreementId
```

Comment:
Bruce and I talked about all the code in this section and he already confirmed that every change was correct.